src/main/java/model/EventModel.java

```diff
@@ -0,0 +1,47 @@
+package model;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import dao.DaoFactory;
+
+public class EventModel {
+	
+	private DaoFactory daoFactory;
+	
+	public EventModel() {
+		this.daoFactory = new DaoFactory();
+	}
+	
+	public ArrayList<HashMap<String, Object>> getSportsCitiesList(String object) {
+		ResultSet list = null;
+		if(object == "city") {
+			list = daoFactory.getEventDao().getCities();		
+		}
+		if(object == "sport") {
+			list = daoFactory.getEventDao().getSports();
+		}
+		ArrayList<HashMap<String, Object>> result = getResultList(list, object);
+		return result;
+	}
+	
+	private ArrayList<HashMap<String, Object>> getResultList(ResultSet data, String object) {
+		ArrayList<HashMap<String, Object>> result = new ArrayList<HashMap<String, Object>>();
+		if (data != null) {
+			try {
+				while(data.next()) {
+					HashMap<String, Object> map = new HashMap<String, Object>();
+					map.put("id_" + object, data.getInt(1));
```

Comment:
1. Try to avoid using completely generic names for variables like object as it can easily lead to a confusing code.
2.  object string is unnecessary, api users will always know which id they are getting if they call method getsports or method getcities.
3. HashMap can probably be <Integer, String>